examples/server_profile.tf

```diff
@@ -10,83 +10,56 @@ provider "oneview" {
 data "oneview_scope" "scope" {
   name = "testing"
 }
-/*
-# Creation of Server Profile without template
-resource "oneview_server_profile" "SP" {
-  name               = "TestSP2"
-  hardware_name      = "0000A66101, bay 3"
+
+# Creates a server profile from server profile tempalte
+
+resource "oneview_server_profile" "SP2" {
+  name               = "TestSP"
+  hardware_name      = "Synergy-Encl-2, bay 8"
   type               = "ServerProfileV12"
-  enclosure_group    = "EG"
-  bios_option {
-    manage_bios = "true"
-    overridden_settings {
-      id    = "TimeFormat"
-      value = "Utc"
-    }
-  }
-  boot_order = ["HardDisk"]
+  enclosure_group    = "SYN03_EC"
+  initial_scope_uris = [data.oneview_scope.scope.uri]
+  template           = "TestServerProfileTemplate"
+}
+
+# Creation of Server Profile without template
+
+resource "oneview_server_profile" "SP2" {
+  name            = "TestSP2"
+  hardware_name   = "0000A66101, bay 5"
+  type            = "ServerProfileV12"
+  enclosure_group = "EG"
+  boot_order      = ["HardDisk"]
   boot_mode {
-    manage_mode     = "true"
+    manage_mode     = true
     mode            = "UEFIOptimized"
     pxe_boot_policy = "Auto"
   }
-  local_storage {
-    controller {
-      device_slot              = "Embedded"
-      drive_write_cache        = "Unmanaged"
-      initialize               = "true"
-      mode                     = "RAID"
-      predictive_spare_rebuild = "Unmanaged"
-      logical_drives {
-        accelerator         = "Unmanaged"
-        bootable            = "true"
-        drive_technology    = "SasHdd"
-        name                = "TestLd"
-        num_physical_drives = 2
-        raid_level          = "RAID1"
+  connection_settings {
+    connections {
+      id            = 1
+      name          = "Deployment Network A"
+      function_type = "Ethernet"
+      network_uri   = "/rest/ethernet-networks/728af64d-c9aa-4287-a331-ba4c6654dd15"
+      port_id       = "Mezz 3:1-a"
+      boot {
+        priority           = "Primary"
+        ethernet_boot_type = "PXE"
       }
     }
   }
-  initial_scope_uris = [data.oneview_scope.scope.uri]
 }
-*/
 
 # Updating Server profile
-resource "oneview_server_profile" "SP" {
-  name                 = "TestSP_Renamed"
-  hardware_name        = "0000A66101, bay 3"
+resource "oneview_server_profile" "SP2" {
+  name                 = "TestSP_Renamed_withCon2"
+  hardware_name        = "0000A66101, bay 5"
   type                 = "ServerProfileV12"
   enclosure_group      = "EG"
-  server_hardware_type = "SY 660 Gen9 1"
-  bios_option {
-    manage_bios = "true"
-    overridden_settings {
-      id    = "TimeFormat"
-      value = "Utc"
-    }
-  } 
-  local_storage {
```

Comment:
why did you reduce the server profile structure..?

Reply:
updated.